llama-index-integrations/vector_stores/llama-index-vector-stores-duckdb/pyproject.toml

```diff
@@ -0,0 +1,50 @@
+[build-system]
+requires = ["poetry-core"]
+build-backend = "poetry.core.masonry.api"
+
+[tool.codespell]
+check-filenames = true
+check-hidden = true
+# Feel free to un-skip examples, and experimental, you will just need to
+# work through many typos (--write-changes and --interactive will help)
+skip = "*.csv,*.html,*.json,*.jsonl,*.pdf,*.txt,*.ipynb"
+
+[tool.mypy]
+disallow_untyped_defs = true
+# Remove venv skip when integrated with pre-commit
+exclude = ["_static", "build", "examples", "notebooks", "venv"]
+ignore_missing_imports = true
+python_version = "3.8"
+
+[tool.poetry]
+name = "llama-index-vector-stores-duckdb"
+version = "0.1.0"
+description = "llama-index vector_stores duckdb integration"
+authors = ["Adithya Krishnan <me@krishadi.com>"]
```

Comment:
Can you also please add yourself as a maintainer like [this](https://github.com/run-llama/llama_index/blob/8431e7baaeb8155c33a7db328fce79bd7d533feb/llama-index-packs/llama-index-packs-agents-llm-compiler/pyproject.toml#L26). Please use your github username.

Reply:
fixed this.